src/hip_hcc.cpp

```diff
@@ -2481,29 +2451,6 @@ bool ihipStream_t::locked_copy2DAsync(void* dst, const void* src, size_t width,
     return retStatus;
 }
 
-//-------------------------------------------------------------------------------------------------
-//-------------------------------------------------------------------------------------------------
-// Profiler, really these should live elsewhere:
-hipError_t hipProfilerStart() {
-    HIP_INIT_API(hipProfilerStart);
-#if COMPILE_HIP_ATP_MARKER
-    amdtResumeProfiling(AMDT_ALL_PROFILING);
-#endif
-
```

Comment:
Again maybe we want to retain the API as a stub similar to hipCtxCreate so that applications don't need to be modified just to be able to build them again.